tests/MyGame/Example/Monster.cs

```diff
@@ -203,6 +205,27 @@ public struct Monster : IFlatbufferObject
     }
     return null;
   }
+
+  public static Monster? __lookup_by_key(Monster obj, int vectorLocation, string key, ByteBuffer bb) {
```

Comment:
Why not make this the only version of this method, and have the accessor function add `new Monster()` as the first argument for the original function?

Reply:
I didn't do it this way because the method __lookup_by_key(..) is public, changing its signature would make it non backward-compatible. If you don't think that's an issue, I will implement the change.

I also thought about just calling the overloaded method from the original __lookup_by_key(..), but since this new method could create new garbage in a case that didn't before (i.e. it would always creates a new Monster() even when the method returns null), I didn't want to make a decision on it.

What's your preference on how I should handle this?

Reply:
I don't think so. Methods starting with `__` are not intended to be used as part of the API, and are sometimes public because they need to be accessed by internal code.. or just because whoever implemented forgot to make them private :(

So maybe the original method can pass `null` and `__lookup_by_key` can check for it? That's an additional conditional, but I think that is acceptable given the complexity of the method.

Reply:
Ok, implemented with the null check.